backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -281,4 +281,42 @@ public UserInfo getCurrentUserInfo() {
             })
         .collect(Collectors.toList());
   }
+
+  @AuthorizationConfiguration.RequirePermissionManageUsers
+  public UserInfo getUserInCurrentOrg(final UUID userId) {
```

Comment:
There is a lot of duplication in this method of code in `getUsersInCurrentOrg()`, which will be removed in the second PR for this ticket.